### PR TITLE
Feature/change playlist

### PR DESCRIPTION
**Pull request by elizeugba**
I finally splitted the features, now I'm pulling the change playlist. I tested it on Windows and my smartphone (Android).

### CODE REVIEW

**Comment by brenonaraujo** [comment]
Damn, I really would like has this feature, seems pretty nice!

**Comment by gustavomerini** [comment]
+1

**Comment by iagolaguna** [comment]
Sorry for the late guys, @elizeugba you forget to put the command in the commands in package.json and your feature isn't working, please fix this, and have some glitch because the owner of the playlist is showing as `null` look this too.

File: src/commands/commands.js [hunk 1]
```diff
@@ -131,6 +131,30 @@ async function handler (error, callback = () => Promise.resolve()) {
   }
 }
 
+async function getUserPlaylists () {
+  const { data: userPlaylistsData } = await axios.get(`${USER_PLAYLISTS_URL}`, {})
+  const basicPlaylistsData = userPlaylistsData.items.map(playlist =>
+    Object.assign({
+      description: `Owner: ${playlist.owner.display_name}`,
```
Comment (iagolaguna) [inline src/commands/commands.js:138]: if Owner doesn't exist will you show `null` for the user ?

File: src/extension.js [hunk 2]
```diff
@@ -112,30 +112,37 @@ const buttonsInfo = [
...
+    priority: 13,
     tooltip: 'Previous',
     buttonCommand: 'vscodefy.previous'
+  },
+  {
+    id: 'change-playlist',
+    text: ' $(list-unordered) ',
+    priority: 14,
+    tooltip: 'Change playlist',
+    buttonCommand: 'vscodefy.change_playlist'
```
Comment (iagolaguna) [inline src/extension.js:145]: please fix `change_playlist` to `changePlaylist` to maintain the pattern

File: src/extension.js [hunk 3]
```diff
@@ -171,5 +178,9 @@ const commandsRegistered = [
   {
     command: 'vscodefy.logout',
     action: logout
+  },
+  {
+    command: 'vscodefy.change_playlist',
```
Comment (iagolaguna) [inline src/extension.js:183]: here the same

**Review by iagolaguna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**